lib/src/main/java/com/dynatrace/file/util/FilePollerFactory.java

```diff
@@ -27,19 +27,18 @@ class FilePollerFactory {
   private FilePollerFactory() {}
 
   /**
-   * Creates the default {@link FilePoller} based on the current OS.
+   * Creates the default {@link FilePoller}.
    *
-   * @implNote On Linux and Windows, * the poll mechanism is based on the {@link
-   *     java.nio.file.WatchService WatchService}. For macOS, * where there is no native
-   *     implementation for the {@link java.nio.file.WatchService WatchService} * bindings, this
-   *     method provides a {@link PollBasedFilePoller}, which polls the file of interest *
-   *     periodically (according to the {@code pollInterval}).
```

Comment:
Should we keep this part? For `getDefault` it's still accurate.

Reply:
IMO this comment is a bit too rigid in explaining what the method does. It's marked as implementation note not but it could be taken that this will stay like this forever by someone reading it.